src/reducers/reducer-chart.js

```diff
@@ -0,0 +1,18 @@
+import { CREATE_NEW_ROW } from "../actions";
+
+const chartReducer = function (state= [], action) {
+  switch(action.type) {
+    
+    case CREATE_NEW_ROW:
+      if (action.payload){
+        return [action.payload, ...state];
```

Comment:
I'd do `[...state, action.payload]` here so that the newest city is at the end of the array